patients/repo.go

```diff
@@ -1166,6 +1167,9 @@ func PatientsToTideResult(patientsList []*Patient, period string, exclusions *[]
 	return categoryResult
 }
 
+const TideReportPatientLimit = 250
+const TideReportNoDataPatientLimit = 100
```

Comment:
Is this used anywhere?

Reply:
> Is this used anywhere?

Uhhh... Ignore that force push, yes, of course it's used. :D